tests/end_to_end/features/steps/serving.py

```diff
@@ -0,0 +1,83 @@
+from behave import *
+import requests
+
+    # And I apply "snowflake_config.py" file with a "hosted" "insecure" CLI
+    # And I create a "hosted" "insecure" client for "localhost:7878" (should be done)
+    # And I apply the "serve_batch_features.py" file
+    # Then I can iterate through the rows of the batch feature table
+    # And I can get a list containing the entity name and a tuple with all the features
+
+@when("I register a Snowflake and with a {database} and {schema}")
+def step_impl(context, database, schema):
+    context.snowflake_name = "test_snowflake"
+    try:
+        context.snowflake = context.featureform.register_snowflake(
+            name = context.snowflake_name,
+            description = "Offline store",
+            team = context.snowflake_team,
+            username = context.snowflake_username,
+            password = context.snowflake_password,
+            account = context.snowflake_account,
+            organization = context.snowflake_organization,
+            database = database,
+            schema = schema,
+        )
+    except Exception as e:
+        context.exception = e
+
+@when('I register the tables from the database')
+def step_impl(context):
+    context.boolean_table = context.snowflake.register_table(
+    name="boolean_table",
+    table= "featureform_resource_feature__8a49dead-41a6-48c39ee7-7ed75d783fe4__0efdd949-f967-4796-8ed3-6c6f736344e8",
+    )
+    context.number_table = context.snowflake.register_table(
+    name="number_table",
+    table="featureform_resource_feature__f4d42278-0889-44d7-9928-8aef22d23c16__6a6e8ff4-8a8f-4217-8096-bb360ae1e99b",
+    )
+
+
+@then("I serve batch features")
+def step_impl(context):
+    try:
+        context.iter = context.client.iterate_feature_set(("table1_feature", "variant_628138"), ("table2_feature", "variant_628138"), ("table3_feature", "variant_628138"), ("table4_feature", "variant_628138"))
```

Comment:
Should change these to use `("table1_feature", ff.get_run())` otherwise these will conflict when running a second time